AndroidProducts.mk

```diff
@@ -3,4 +3,6 @@ PRODUCT_MAKEFILES := \
 	$(LOCAL_DIR)/aosp_marlin_svelte.mk \
 	$(LOCAL_DIR)/aosp_sailfish.mk \
 	$(LOCAL_DIR)/lineage_marlin.mk \
-	$(LOCAL_DIR)/lineage_sailfish.mk
+	$(LOCAL_DIR)/lineage_sailfish.mk \
```

Comment:
Nit: we can use the following snippet to add maru makefiles without modifying origin statement directly:

```Makefile
PRODUCT_MAKEFILES += \
    $(LOCAL_DIR)/maru_mariln.mk \
    $(LOCAL_DIR)/maru_sailfish.mk \
```